src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1448,6 +1449,31 @@ void goto_instrument_parse_optionst::instrument_goto_program()
       throw 0;
   }
 
+  if(
+    cmdline.isset("replace-function-body") ||
+    cmdline.isset("generate-function-body"))
+  {
+    if(
+      cmdline.isset("replace-function-body") &&
+      cmdline.isset("generate-function-body"))
+    {
+      throw "can only use one of --replace-function-body "
+        "or --generate-function-body";
+    }
+    bool only_generate = cmdline.isset("generate-function-body");
+    std::regex function_regex = std::regex(
+      only_generate ? cmdline.get_value("generate-function-body")
+                    : cmdline.get_value("replace-function-body"));
+    status() << "Replacing function bodies" << eom;
```

Comment:
Should this method say `"Generating function bodies"` if `cmdline.get_value("generate-function-body")`